op_builder/mlu/fused_adam.py

```diff
@@ -5,7 +5,11 @@
 # DeepSpeed Team
 
 from .builder import MLUOpBuilder
-import torch
+
+try:
```

Comment:
Can we import `torch` in `multi_tensor_adam` if it is the only place where you need torch?
Then you can throw a more specific error when torch is unavailable and `multi_tensor_adam` is called.